src/mudlet.cpp

```diff
@@ -2270,7 +2270,7 @@ void mudlet::slot_show_help_dialog_forum()
 
 void mudlet::slot_show_help_dialog_irc()
```

Comment:
FTR This (currently unused slot) method is supposed to open a web-based IRC client within the end-user's default web-browser. This means it is external to the Mudlet application - which is a different arrangement to the only option we now/currently provide using an internal to Mudlet IRC client (a Qt-based one provided by the communi library which we include a copy of in our source code)...